lametro/templates/500.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
+
+{% load staticfiles %}
+
+{% block title %}Internal server error{% endblock %}
+
+{% block full_content %}
+
+    <div class="row-fluid">
+        <div class="col-sm-12">
+            <h1>Internal Server Error</h1>
+            <p>Sorry for the temporary inconvenience! Please visit <a href="https://metro.legistar.com/calendar.aspx">https://metro.legistar.com/calendar.aspx</a> to access Metro Board Meetings &amp; Agendas while we repair the issue.</p>
+            <p><a href="/">&laquo; go back to the home page</a></p>
```

Comment:
Do we want to add `target="_blank"`? So that we do not redirect the user away from Councilmatic?